lib/scylla_cloud.py

```diff
@@ -21,6 +21,21 @@
 from abc import ABCMeta, abstractmethod
 from lib.scylla_cloud_io_setup import aws_io_setup, gcp_io_setup, azure_io_setup
 
+
+def out(cmd, shell=True, timeout=None, encoding='utf-8', ignore_error=False):
+    res = run(cmd, capture_output=True, shell=shell, timeout=timeout, check=False, encoding=encoding)
+    if not ignore_error and res.returncode != 0:
+        print(f'Command \'{cmd}\' returned non-zero exit status: {res.returncode}')
```

Comment:
Not critical but maybe here we can catch the `CalledProcessError` exception and use the `CalledProcessError.output`

Reply:
switched to catching CalledProcessError exception.